packages/framework-core/src/decorators/global-event-handler.ts

```diff
@@ -0,0 +1,24 @@
+import { EventHandlerInterface, BoosterConfig } from '@boostercloud/framework-types'
+import { Booster } from '../booster'
+
+export const GLOBAL_EVENT_HANDLER = 'GLOBAL_EVENT_HANDLER'
+
+export function GlobalEventHandler(): <TEventHandler extends EventHandlerInterface>(
+  eventHandlerClass: TEventHandler
+) => void {
+  return (eventHandlerClass) => registerGlobalEventHandler(eventHandlerClass)
+}
```

Comment:
Two things here: 
1. This decorator doesn't receive anything as parameter, so it doesn't have to be a function that return the decorator. It can be the decorator itself.
2. Taking into account the previous comment, we can delete the function `registerGlobalEventHandler` and call the one that is in `event-handler.ts`. This way we avoid repeating the logic.

This is the suggested code with both changes applied:

```suggestion
export function GlobalEventHandler<TEventHandler extends EventHandlerInterface>(
  eventHandlerClass: TEventHandler
): void {
  // We would need to do the function `registerEventHandler` exported
  registerEventHandler(GLOBAL_EVENT_HANDLER, eventHandlerClass) 
}
```